grafana/dashboards/overview.json

```diff
@@ -1329,8 +1535,8 @@
       {
         "allValue": null,
         "current": {
-          "text": "http://localhost:4000",
-          "value": "http://localhost:4000"
+          "text": "https://teslamate.schmidt-schmitten.com",
```

Comment:
Please revert this

Reply:
Not sure, whether the https://teslamate.schmidt-schmitten.comis completely out now...